Applications/Games/Heroes of the Storm/Online/script.js

```diff
@@ -0,0 +1,21 @@
+include(["Engines", "Wine", "QuickScript", "OnlineInstallerScript"]);
+include(["Engines", "Wine", "Verbs", "vcrun2015"]);
+include(["Engines", "Wine", "Verbs", "corefonts"]);
+
+new OnlineInstallerScript()
+    .name("Heroes of the Storm")                       
+    .editor("Blizzard")                               
+    .applicationHomepage("http://eu.battle.net/heroes/")  
+    .author("ImperatorS79")                     
+    .url("https://eu.battle.net/download/getInstaller?os=win&installer=Heroes-of-the-Storm-Setup.exe")                       
+    .category("Games")                          
+    .executable("Heroes of the Storm.exe")
+    //The checksum is different each time you download
```

Comment:
@plata I'm not sure, but I think, that the checksum is currently ignored, right?
If we use the checksum later on to validate the downloaded file, we shouldn't forget to add the possibility that the checksum is unknown.

Reply:
Without checking, I think that the checksum is optional. So if you leave it aways, it's ignored. I think what @ImperatorS79 did is correct.